torchbenchmark/models/stable_diffusion/__init__.py

```diff
@@ -23,12 +23,12 @@ class Model(HuggingFaceAuthMixin, BenchmarkModel):
 
     def __init__(self, test, device, jit=False, batch_size=None, extra_args=[]):
         HuggingFaceAuthMixin.__init__(self)
-        super().__init__(test=test, device=device, jit=jit,
+        BenchmarkModel.__init__(self,test=test, device=device, jit=jit,
```

Comment:
Why is this change needed?

Reply:
`super()` was referencing the mixin so this was failing, I think I can fix this by flipping the order of inherited classes though

Reply:
> `super()` was referencing the mixin so this was failing, I think I can fix this by flipping the order of inherited classes though

Yes please do that, thanks!